rtt_rosparam/include/rtt_rosparam/rosparam.h

```diff
@@ -53,31 +59,36 @@ namespace rtt_rosparam {
       this->addOperationCaller(getAbsolute);
       this->addOperationCaller(getPrivate);
       this->addOperationCaller(getComponentPrivate);
+      this->addOperationCaller(getComponentRelative);
 
       this->addOperationCaller(set);
       this->addOperationCaller(setParam);
       this->addOperationCaller(setRelative);
       this->addOperationCaller(setAbsolute);
       this->addOperationCaller(setPrivate);
       this->addOperationCaller(setComponentPrivate);
+      this->addOperationCaller(setComponentRelative);
     }
 
     typedef enum  {
       RELATIVE, //! Relative resolution:  "name" -> "name"
       ABSOLUTE, //! Absolute resolution:  "name" -> "/name"
       PRIVATE,  //! Private resolution:   "name" -> "~name"
-      COMPONENT //! Component resolution: "name" -> "~COMPONENT_NAME/name"
+      COMPONENT_PRIVATE, //! Component resolution: "name" -> "~COMPONENT_NAME/name"
```

Comment:
Maybe the `COMPONENT` constant should stay, for backwards compatibility. We could solve this by adding
```
COMPONENT = COMPONENT_PRIVATE
```
at the end of the enumerator list.